src/ray/protobuf/serve.proto

```diff
@@ -131,6 +135,11 @@ message EndpointSet {
   map<string, EndpointInfo> endpoints = 1;
 }
 
+message ActorNameList {
```

Comment:
document why do we need this (instead of actor handles)

Reply:
Document is added.